src/main/java/dev/slimevr/websocketapi/WebSocketVRBridge.java

```diff
@@ -93,17 +93,20 @@ public void onMessage(WebSocket conn, String message) {
 			JSONObject json = new JSONObject(message);
 			if (json.has("type")) {
 				switch (json.optString("type")) {
-					case "pos":
+					case "pos" -> {
 						parsePosition(json, conn);
 						return;
-					case "action":
+					}
+					case "action" -> {
 						parseAction(json, conn);
 						return;
-					case "config": // TODO Ignore it for now, it should only
-									// register HMD in our test case with id
+					}
+					case "config" -> { // TODO Ignore it for now, it should only
+						// register HMD in our test case with id
 						// 0
```

Comment:
Weirdly split comment
```suggestion
					case "config" -> { // TODO Ignore it for now, it should only
						// register HMD in our test case with id 0
```

Reply:
this one is weird, [look at the original](https://github.com/SlimeVR/SlimeVR-Server/blob/main/src/main/java/dev/slimevr/websocketapi/WebSocketVRBridge.java#L102)